src/lib/Libifl/pbs_loadconf.c

```diff
@@ -255,6 +257,58 @@ parse_config_line(FILE *fp, char **key, char **val)
 	return ret;
 }
 
+
+/**
+ * @brief
+ * parse_psi - parses the PBS_SERVER_INSTANCES
+ *    
+ * @param[in] conf_value  configuration value set in pbs.conf
+ * 
+ * @return int
+ * @retval -1, For error
+ * @retval 0, For success
+ */
+
+int
+parse_psi(char *conf_value)
+{
+	char **list;
+	int i;
+
+	free(pbs_conf.psi);
+
+	list = break_comma_list(conf_value);
+	if (list == NULL)
+		return -1;
+
+	for (i = 0; list[i] != NULL; i++)
+		;
+
+	if (!(pbs_conf.psi = calloc(i, sizeof(pbs_server_instance)))) {
+		fprintf(stderr, "Ran out of memory parsing configuration %s", conf_value);
```

Comment:
How about `Out of memory while parsing PBS_SERVER_INSTANCES: %s`?

Reply:
I agree, addressed it.